src/fsharp/FSharp.Compiler/FSharp.Compiler.fsproj

```diff
@@ -11,7 +11,7 @@
     <OutputType>Library</OutputType>
     <AssemblyName>FSharp.Compiler</AssemblyName>
     <DefineConstants>EXTENSIONTYPING;COMPILER;INCLUDE_METADATA_READER;INCLUDE_METADATA_WRITER;EXTENSIBLE_DUMPER;$(DefineConstants)</DefineConstants>
-    <NoWarn>$(NoWarn);62;9</NoWarn>
+    <NoWarn>$(NoWarn);62;9;3180</NoWarn>
```

Comment:
I think you get the error, because warning 3180 has to explicitly switched on to be reported.
[FSharpSource.Settings.targets](https://github.com/Microsoft/visualfsharp/blob/master/src/FSharpSource.Settings.targets#L29) switches it on only for debug builds and I think the file is only included in some builds.